front/src/components/boxs/device-in-room/device-features/ColorDeviceFeature.jsx

```diff
@@ -64,7 +64,7 @@ class ColorDeviceType extends Component {
 
   componentDidMount() {
     const deviceLastValue = this.props.deviceFeature.last_value;
-    const color = deviceLastValue === null ? undefined : `#${intToHex(deviceLastValue)}`;
+    const color = !deviceLastValue ? undefined : `#${intToHex(deviceLastValue)}`;
```

Comment:
Mmm wait, does it means that when the color is 0, it shows an `undefined`color ? Shouldn't it be the black hexadecimal color?

Reply:
The color pocker doesn't have black color, it causes trouble, and I don't think your light has any black color too.
Black means switched off, it is not really a color.

Reply:
True!